courseworkA/Markdown report template assignment A .Rmd

```diff
@@ -247,66 +256,234 @@ Describe the statistical test you suggest to care out on the collected data
 <!-- Compare sentiments of celebrity pairs and provide a brief interpretation (e.g. CIs)  -->
 
 
-<!-- ## Question 2 - Website visits (between groups - Two factors) -->
+## Question 2 - Website visits (between groups - Two factors)
 
-<!-- ### Conceptual model -->
+### Conceptual model
 <!-- Make a conceptual model underlying this research question -->
 
-<!-- ### Visual inspection -->
+```{r concept_diag, echo=FALSE, fig.cap="\\label{fig:concept_diag}The conceptual model underlying the research question.", out.width = '100%'}
+knitr::include_graphics("img/concept_diag.png")
+```
+
+### Visual inspection
 <!-- Graphically examine the variation in page visits for different factors levels (e.g. histogram, density plot etc.)  -->
 
 
-<!-- ```{r} -->
-<!-- #include your code and output in the document -->
-<!-- ``` -->
+```{r}
+filepath <- ("webvisit0.csv")
+data <- read.csv(file=filepath, header=TRUE)
+
+# changing dtype of the factors
+data$portal = as.factor(data$portal)
+data$version = as.factor(data$version)
+
+# Function to calculate the mean and the standard deviation for each factor group
+
+data_summary <- function(data, varname, groupnames){
+  require(plyr)
+  summary_func <- function(x, col){
+    c(mean = mean(x[[col]], na.rm=TRUE),
+      sd = sd(x[[col]], na.rm=TRUE))
+  }
+  data_sum<-ddply(data, groupnames, .fun=summary_func,
+                  varname)
+  data_sum <- rename(data_sum, c("mean" = varname))
+ return(data_sum)
+}
+
+df3 <- data_summary(data, varname="pages", 
+                    groupnames=c("version", "portal"))
+
+p <- ggplot(df3, aes(x=version, y=pages, fill=portal)) + 
+   geom_bar(stat="identity", position=position_dodge()) +
+  geom_errorbar(aes(ymin=pages-sd, ymax=pages+sd), width=.2,
+                 position=position_dodge(.9))
+  
+p + scale_fill_brewer(palette="Paired") + theme_minimal()
+
+# Creating subsets of data for each combination of factors
+subset00 <- subset(data, version == '0' & portal == '0')
+subset01 <- subset(data, version == '0' & portal == '1')
+subset10 <- subset(data, version == '1' & portal == '0')
+subset11 <- subset(data, version == '1' & portal == '1')
+
+```
 
```

Comment:
maybe say something about the figure here?

Reply:
Notable observations from the bar-plot demonstrating the mean and standard deviation of the page visits are that the mean page visits across both the versions for the 0 - portal entries (Consumer) are almost the same but vary significantly for the 1 - portal entries (Company).